Sitemap/Voter.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Symfony\Cmf\Bundle\SeoBundle\Sitemap;
+
+/**
+ * A chain as a decorator for all tagged voter services.
+ *
+ * @author Maximilian Berghoff <Maximilian.Berghoff@mayflower.de>
+ */
+class Voter extends AbstractChain implements VoterInterface
```

Comment:
uhhh, ohhh this would make my abstraction for the chain useless, as there will be one chain afterwards.